__tests__/spec-only.js

```diff
@@ -0,0 +1,56 @@
+'use strict'
+
+const rule = require('../rules/spec-only')
+const { RuleTester } = require('./rule-tester')
+const ruleTester = new RuleTester()
+
+ruleTester.run('spec-only', rule, {
+  valid: [
+    'Promise.resolve()',
+    'Promise.reject()',
+    'Promise.all()',
+    'Promise.race()',
```

Comment:
Could you please add a test for `withResolvers`? I think it's probably not listed in `promise-statics` yet.

Reply:
Sure, I added a commit for this if you can review. Note that by adding it to the promise-statics matrix, I had to remove it explicitly from the `isPromise` detection since `withResolvers` does not directly return a `Promise` (it returns it on the `promise` property).